modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -53,47 +63,47 @@ contains the `install-config.yaml` file you created.
 Because you create your own compute machines later in the installation process,
 you can safely ignore this warning.
 
+ifdef::aws,gcp[]
 . Remove the Kubernetes manifest files that define the control plane machines:
 +
 ----
 $ rm -f openshift/99_openshift-cluster-api_master-machines-*.yaml
 ----
 +
 By removing these files, you prevent the cluster from automatically generating control plane machines.
-
-ifdef::aws[]
-. Remove the Kubernetes manifest files that define the worker machines:
-+
-----
-$ rm -f openshift/99_openshift-cluster-api_worker-machineset-*
-----
-+
-Because you create and manage the worker machines yourself, you do not need
-to initialize these machines.
-endif::[]
+endif::aws,gcp[]
 
 ifdef::gcp[]
 . Optional: If you do not want the cluster to provision compute machines, remove
 the Kubernetes manifest files that define the worker machines:
+endif::gcp[]
+ifdef::aws[]
+. Remove the Kubernetes manifest files that define the worker machines:
+endif::aws[]
+ifdef::aws,gcp[]
 +
 ----
 $ rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
 ----
 +
 Because you create and manage the worker machines yourself, you do not need
 to initialize these machines.
+endif::aws,gcp[]
 
-. Optional: Currently, emptying the compute pools makes control plane machines
-schedulable. However, due to a
-link:https://github.com/kubernetes/kubernetes/issues/65618[Kubernetes limitation],
-router Pods running on control plane machines will not be reachable by the
-ingress load balancer.
+. Modify the `manifests/cluster-scheduler-02-config.yml` Kubernetes manifest file to prevent Pods from being scheduled on the control plane machines:
```

Comment:
@jianlinliu, in the original GCP PR, this was marked as an optional step. Will you please double-confirm that we always need to change this setting on GCP?

Reply:
https://github.com/openshift/openshift-docs/pull/17043

Reply:
Per my test result, I do not think it is a optional step when we are guiding user set worker replica to 0. Now I think everywhere keep consistent now, it is a MUST.

Reply:
@adellape, FYI ^